packages/create/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@clusterio/create",
+  "description": "Installer for clusterio",
+  "author": "Hornwitser",
+  "license": "MIT",
+  "version": "2.0.0-alpha.0",
```

Comment:
Are you counting versions seperately for the create script?

Reply:
No, I uploaded it as this version to test it, the next release will make it go in sync with the rest of the packages.